perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/EventDispatcherImpl.java

```diff
@@ -308,14 +308,21 @@ protected MessageBeans resolveMessage(String msg, Integer idOfMessage) throws In
 
 		//Debug information to check parsing of message.
 		MessageBeans beans = new MessageBeansImpl();
+
+		boolean shouldntAddVo = msg.matches("[\\d\\D]*Group:[\\d\\D]*created in Vo:[\\d\\D]*");
```

Comment:
I don't think we need ``[\\d\\D]*`` at the start, since relevant messages about groups always start with ``Group ...``.

Generally, I would prefer, if we keep consistent regex usage. If you look at the code, we match strings using ``Pattern`` and ``Matcher`` with modifier ``Pattern.DOTALL``, so you can use ``(.*)`` to match all content of the group/vo object including newlines (which can be in group description).